frontend_v2/src/app/services/endpoints.service.ts

```diff
@@ -309,7 +309,7 @@ ${phase}/submission?participant_team__team_name=${participantTeamName}`;
   }
 
   /**
-   * Challenge Submission Counts
+   * Challenge Submission Counts of the participant Team 
```

Comment:
Can we please revert this change as it's required for this PR

Reply:
We were using this url in the submissions page, but this was confusing whether it was for all submissions or submissions for a particular team that's why I have modified it to make it more descriptive.